connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/application/ReplE2ESuite.scala

```diff
@@ -274,6 +274,12 @@ class ReplE2ESuite extends RemoteSparkSession with BeforeAndAfterEach {
         |  collect()
       """.stripMargin
     val output = runCommandsInShell(input)
-    assertContains("Array[MyTestClass] = Array(MyTestClass(1), MyTestClass(3))", output)
+    if (Properties.versionNumberString.startsWith("2.13")) {
```

Comment:
Alternatively we morph the result into something that is the same for both scala versions.

Reply:
For example, remove `value =` from `out` ?